test/test_nn.py

```diff
@@ -13840,8 +13891,8 @@ def check(x, args, message):
                 model(torch.tensor(x, device=device, dtype=dtype))
 
         # Pooling args: (kernel_size, stride, padding, dilation, return_indices, ceil_mode)
-        check(0, (1,), "input tensor must have 2 or 3 dimensions but got 0")
-        check([], (1,), "input tensor must have 2 or 3 dimensions but got 1")
+        check(0, (1,), "Expected 2D or 3D input tensor, but got 0")
+        check([], (1,), "Expected 2D or 3D input tensor, but got 1")
```

Comment:
Similarly here, removing the " 1" should get the check passing.

Reply:
Extremely sorry for the mistake. Have updated it.